test/fixtures/code_formatting/swift

```diff
@@ -0,0 +1,121 @@
+//
+//  PhoneNumber.swift
+//  PhoneNumber
+//
+//  Created by Tom Holland on 8/13/16.
+//  Copyright © 2016 Tom Holland. All rights reserved.
```

Comment:
Taking code other peoples (non open source) code is a big no-no.
In any case it would be better to recreate the issue using a the minimal possible example code.